### DISCUSSION TITLE
Might be useful for the new scheduler Gsoc 2021 students are working on

### DISCUSSION THREAD

**Discussion by alexandreelise** [joomla/joomla-cms#35489] [2. Cold Features]
Might be useful for the new scheduler Gsoc 2021 students are working on

[Php Cron Scheduler](https://github.com/peppeocchi/php-cron-scheduler)

@bembelimen

**Comment by ditsuke** [comment]
Thanks for sharing @alexandreelise. I'll have a look at it again and comb for ideas, already spot a few 😃

**Reply by alexandreelise** [reply]
No problem @ditsuke . Have a great day. Happy coding!
